packages/SwingSet/src/kernel/kernel.js

```diff
@@ -180,6 +188,11 @@ export default function buildKernel(kernelEndowments) {
     insistCapData(data);
     const p = getResolveablePromise(kpid, vatID);
     const { subscribers, queue } = p;
+    let idx = 0;
+    for (const dataSlot of data.slots) {
+      kernelKeeper.incrementKernelPromiseRefCount(dataSlot, `fulfill|s${idx}`);
```

Comment:
What would you think about making `kernelKeeper.fulfillKernelPromiseToData` be responsible for updating the refcount on the slots, instead of doing it here? I'm kind of on the fence as to which would be most clear.

In any case, `fulfillToPresence` should probably get an `increment` too, if we make this more general and not so promise-specific, even though that increment will get ignored for now because we only refcount promises, not presences.